src/main/java/com/gmail/goosius/siegewar/events/SiegeWarStartEvent.java

```diff
@@ -60,4 +62,8 @@ public Nation getNation() {
 	public Town getTargetTown() {
 		return targetTown;
 	}
+
+    public String getMessage() {
+        return LegacyComponentSerializer.legacySection().deserialize(SiegeController.getGlobalSiegeStartMessage(siege).defaultLocale()).content();
```

Comment:
This even should probably just pass in the message it's already thought up, instead of going back to the SiegeController and re-creating the message new.